spec/unit/provider/network_config/interfaces_spec.rb

```diff
@@ -230,21 +230,37 @@ def fixture_data(file)
 
     describe "writing the auto section" do
       it "should allow at most one section" do
-        content.scan(/^auto .*$/).length.should == 1
+        content.scan(/^auto .+$/).length.should == 1
       end
 
       it "should have the correct interfaces appended" do
-        content.scan(/^auto .*$/).first.should match("auto eth0 lo")
+        content.scan(/^auto .+$/).first.should match("auto eth0 lo")
+      end
+    end
+
+    describe "writing only the auto section" do
+      let(:content) { described_class.format_file('', [lo_provider]) }
+
+      it "should fail if allow-hotplug is present" do
```

Comment:
Would this be a more fitting description?

`should skip the allow-hotplug line`